### DISCUSSION TITLE
Why was an `Output` property added to the `ITestOutputHelper` interface in xUnit v3?

### DISCUSSION THREAD

**Discussion by 0xced** [xunit/xunit#3027] [Question] [answered]
@bradwilson Could you please shed some light on the purpose of this new property?

This question has arisen while discussing https://github.com/testcontainers/testcontainers-dotnet/pull/1165#issuecomment-2359603398

**Comment by bradwilson** [comment, marked as answer]
It was added to give access to the output to extensions; otherwise, the output is only available during the reporting of the test result message.
